tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -453,6 +453,33 @@ def GraphExecutionData(self, run, begin, end, trace_id=None):
             ],
         }
 
+    def GraphInfo(self, run, graph_id):
+        """Get the information regarding a TensorFlow graph.
+
+        Args:
+          run: Name of the run.
+          graph_id: Debugger-generated ID of the graph in question.
+            This information is available in the return values
+            of `GraphOpInfo`, `GraphExecution`, etc.
+
+        Returns:
+          A JSON-serializable object containing the information regarding
+            the TensorFlow graph.
+
+        Raises:
+          NotFoundError if the graph_id is not known to the debugger.
+        """
+        runs = self.Runs()
+        if run not in runs:
+            return None
+        try:
+            graph = self._reader.graph_by_id(graph_id)
```

Comment:
Why is `run` not actually used in the query? Is it needed?

Reply:
Please recall that we currently support only up to one tfdbg2 run in each logdir, as read by `self._reader`. It is identified with the hardcoded run name `__default_debugger_run__`. You can see other public methods of this class do similar checks of `run` just to ensure it exists in the beginning, without using it for actual data look up subsequently.